Makefile

```diff
@@ -217,6 +217,21 @@ release-image-name: ## Prints the image name of the release image
 release-image-tag: ## Prints the image tag of the release image
 	@echo $(IMAGE_TAG)
 
+########################
+## e2e                ##
+########################
+
+e2e-setup: ## Setup end-to-end tests
+	@cd e2e && npm install
+	@cd e2e && npx playwright install --with-deps
+
+
+e2e-test: ## Run end-to-end tests
+    # make e2e-test APP_NAME=app BASE_URL=http://localhost:3000
+	@:$(call check_defined, APP_NAME, ...)
+	@:$(call check_defined, BASE_URL, ...)
+	@cd e2e && cd $(APP_NAME) && APP_NAME=$(APP_NAME) BASE_URL=$(BASE_URL) npx playwright test $(E2E_ARGS)
+
```

Comment:
these make commands are one way to trigger the tests, but we can also trigger in the e2e folder with the e2e-tests script as well (the latter will use the env set - whether local .env or CI envs)